codegen/src/main/java/com/paypal/selion/elements/HtmlSeLionElement.java

```diff
@@ -53,6 +56,20 @@
     public String toString() {
         return String.format("%s, canHaveParent=%s",super.toString(), canHaveParent);
     }
+    
+    /**
+     * By providing the qualified name of a custom element we can register it to the element array.
+     * Custom elements are inserted before SeLion elements, if you use the same name it will overwrite the existing element.
+     * 
+     * @param element string of the qualified class
+     */
+    public static void registerElement(String element) {
+        List<HtmlSeLionElement> temp = new ArrayList<HtmlSeLionElement>(Arrays.asList(values));
+        
+        temp.add(0, new HtmlSeLionElement(HtmlElementUtils.getPackage(element), HtmlElementUtils.getClass(element), true, false));
+        
+        values = temp.toArray(new HtmlSeLionElement[temp.size()]);
```

Comment:
I have updated my code, in the CodeGeneratorMojo I check if the element is an existing SeLion element. If so it will emit a custom log message to make the user aware of this.